ladybug/_datacollectionbase.py

```diff
@@ -670,7 +670,43 @@ def linspace(start, stop, num):
             return [start]
 
     @staticmethod
-    def histogram(values, bins, hist_range=None, key=None):
+    def arange(start, stop, step):
+        """Return evenly spaced fractional or whole values within a given interval.
+
+        This function acts like the Python range method, but can also account for
+        fractional values. It is equivalent to the numpy.arange function.
+
+        Args:
+            start: Number for inclusive start of interval.
+            stop: Number for exclusive end of interval.
+            step: Number for step size of interval.
+
+        Returns:
+            Generator of evenly spaced values.
+
+        Usage:
+
+        .. code-block:: python
+
+            from BaseCollection import arange
+
+            arange(1, 351, 50)
+            # >> [1, 51, 101, 151, 201, 251, 301]
+        """
+
+        val = start
+
+        if start <= stop:
+            def ineq(a, b): return a < b
```

Comment:
This doesn't feel very PEP8 to me.  I think there should at least be an indent here.

Reply:
So this was automatically done by autopep8 (which the VSCode Python extension kept asking me to install). I had this as a lambda originally and autopep8 rewrote it as the above.

Reply:
Ok. It seems I'm just mistaken here. We can keep it as is.